_posts/en/newsletters/2024-07-12-newsletter.md

```diff
@@ -0,0 +1,122 @@
+This week's newsletter FIXME
+
+## News
+
+*No significant news this week was found in any of our [sources][].  For
+fun, you may want to check out a recent [interesting transaction][].*
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[Testnet4 including PoW difficulty adjustment fix][review club 29775] is
+a PR by [Fabian Jahr][gh fjahr] that introduces Testnet4 as a new test
+network to replace Testnet3 and simultaneously fixes the long-standing
+difficulty adjustment and time warp bugs. It is the result of a
+[discussion on the mailing-list][ml testnet4] and is accompanied by a
+[BIP proposal][bip testnet4].
+
+{% include functions/details-list.md
+  q0="Aside from the consensus changes, what differences do you see
+  between Testnet 4 and Testnet 3, particularly the chain params?"
+  a0="The deployment heights of past soft forks are all set to 1, which
+  means they're active from the beginning. Testnet4 also uses a
+  different port (`48333`) and messagestart, and it has a new genesis
+  block message."
+  a0link="https://bitcoincore.reviews/29775#l-29"
+
+  q1="How does the 20-min exception rule work in Testnet 3? How does
+  this lead to the block storm bug?"
+  a1="If a new block's timestamp is more than 20 minutes ahead of the
+  previous block's timestamp, it is allowed to have minimum
+  proof-of-work difficulty. The next block is subject to the \"real\"
+  difficulty again, unless it too falls under the 20-min exception rule.
+  This exception is made to enable the chain to make progress in an
+  environment of highly variable hashrate. Due to a bug in the
+  `GetNextWorkRequired()` implementation, the difficulty is actually
+  reset (instead of temporarily lowered for just one block) when the
+  last block of a difficulty period is a min-difficulty block."
+  a1link="https://bitcoincore.reviews/29775#l-47"
+
+  q2="Why was the time warp fix included in the PR? How does the time
+  warp fix work?"
+  a2="A [time warp][topic time warp] attack allows an attacker to
+  significantly alter the block production rate, so it makes sense to
+  fix this alongside the min-difficulty bug. Since it is also part of
+  the [consensus cleanup soft fork][topic consensus cleanup],
+  test-running the fix in Testnet4 first provides useful early feedback.
+  This PR fixes the time warp bug by checking that the first block of a
+  new difficulty epoch is not earlier than 2 hours before the last block
+  of the previous epoch."
+  a2link="https://bitcoincore.reviews/29775#l-68"
+
+  q3="What is the message in the Genesis block in Testnet 3?"
+  a3="Testnet 3, as well as all other nets (preceding Testnet 4), have
+  the same well-known genesis block message: \"The Times
+  03/Jan/2009 _Chancellor_ on _brink_ of second bailout for banks\".
```

Comment:
@stickies-v Why are "Chancellor" and "brink" in italics here?

Reply:
Removed in a separate commit so that it can be reverted if the italics were desired.

Reply:
I copied it from some random website instead of from the codebase and it looks like I'm paying the price for it. Thanks for fixing.